src/odemis/driver/test/technolution_test.py

```diff
@@ -446,30 +446,43 @@ def test_clock_VAs(self):
                 1 / clock_freq)
 
     def test_resolution_VA(self):
+        """
+        The setter allows only to enter resolutions which are a whole multiple of 4 times the mppc detector shape
+        """
         min_res = self.EBeamScanner.resolution.range[0][0]
         max_res = self.EBeamScanner.resolution.range[1][0]
 
         # Check if small resolution values are allowed
-        self.EBeamScanner.resolution.value = (min_res + 5, min_res + 5)
-        self.assertEqual(self.EBeamScanner.resolution.value, (min_res + 5, min_res + 5))
+        self.EBeamScanner.resolution.value = (self.MPPC.shape[0] * 4, self.MPPC.shape[0] * 4)
```

Comment:
The minimum for the eff_cell_size is 10. So actually I just noticed, that the minimum for the scanner.resolution VA should be 10 * 8. For the maximum it is already correct. 
`self.MPPC.shape[0] * 4` -> replace with `min_res`

Reply:
Actually the minimum should be 8 * 4 = 32. Do we want to change that? (this gives effective cell sizes of 1)

Reply:
The API yaml file denotes 10 as minimum for the eff_cell_size. That's why I calculate 10 * 8. So 32 doesn't sound right to me.
Then additionally the multiple of 4 rule kicks in. So the minimum must be 12 * 8. But maybe I oversee something.

Reply:
No you are right I will change it to 12 * 8